src/test/scala/cromwell/webservice/CromwellApiServiceIntegrationSpec.scala

```diff
@@ -16,13 +16,13 @@ import scala.concurrent.duration.Duration
 
 class CromwellApiServiceIntegrationSpec extends FlatSpec with CromwellApiService with ScalatestRouteTest with Matchers {
   def actorRefFactory = system
-  val dataAccess = DataAccess()
-  val workflowManager = system.actorOf(WorkflowManagerActor.props(dataAccess, CromwellSpec.BackendInstance))
+  val dataAccess = DataAccess.instance
+  val workflowManager = system.actorOf(WorkflowManagerActor.props(CromwellSpec.BackendInstance))
   val version = "v1"
 
   override protected def afterAll() {
     super.afterAll()
-    Await.result(dataAccess.shutdown(), Duration.Inf)
+    //Await.result(dataAccess.shutdown(), Duration.Inf)
```

Comment:
Delete this entire override. Git's revision control will save it for posterity if we need it again.